versioned/tiered/tiered-impl/src/main/java/org/projectnessie/versioned/impl/condition/UpdateClause.java

```diff
@@ -27,4 +27,13 @@
 
   String toClauseString();
 
+  /**
+   * Default implementation for visitation.
+   * @param visitor the visitor that will be invoked.
+   * @param <T> the type of the returned value.
+   * @return the possibly transformed value resulting from the visitation.
+   */
+  default <T> T accept(UpdateClauseVisitor<T> visitor) {
+    throw new IllegalArgumentException();
```

Comment:
We had gotten advice that we shouldn't use defaults, simply leave as abstract.

Reply:
Done